src/zui/ZUIPersonGridEditCell.tsx

```diff
@@ -0,0 +1,288 @@
+import { Close } from '@mui/icons-material';
+import { makeStyles } from '@mui/styles';
+import { useRouter } from 'next/router';
+import { Msg, PlainMessage, useMessages } from 'core/i18n';
+
+import messageIds from './l10n/messageIds';
+import { usePersonSelect } from './ZUIPersonSelect';
+import { ZetkinPerson } from 'utils/types/zetkin';
+import ZUIAvatar from 'zui/ZUIAvatar';
+
+import {
+  Box,
+  Button,
+  InputBase,
+  List,
+  ListItem,
+  ListSubheader,
+  Paper,
+  Popper,
+  Theme,
+  Typography,
+  useAutocomplete,
+} from '@mui/material';
+import { FC, HTMLAttributes, useState } from 'react';
```

Comment:
I think the imports are a little bit messy. The grouping is usually:

1. External dependencies
2. Our own dependencies
3. `messageIds` on it's own, since it's special

I'm not sure if anyone else cares about this particular grouping, but if there is grouping (like here, where there are three groups of imports) I think they should at least follow some logic.

Reply:
Alright